jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -212,6 +212,14 @@ def step020_install_server(SKIP_SERVER_INSTALL, script_dir, server_era, agent_ve
     }
 }
 
+def step021_clone_and_rsync(script_dir, remote, ref) {
```

Comment:
Are there rules around this numbering scheme (here and the way we tend to do steps in the main CI Jenkins pipelines)? I've always been a little confused by it.

Reply:
No rules, it just mirrors the order in the pipeline itself. I figured right after the server install would be the best spot, but I don't care too much.

Reply:
I was more curious about the intervals, when to increment 1 vs. 5 vs. 10. Always seems sorta arbitrary, which it very well might be :)

Reply:
Oh yea. Totally arbitrary.

Reply:
This *felt* closer to the install 🤷‍♀️